src/attorney.js

```diff
@@ -195,6 +196,20 @@ function applyArchiveConfig (config) {
   }
 }
 
+function applyArchiveFailedConfig (config) {
+  const ARCHIVE_DEFAULT = 60 * 60 * 24 * 7
```

Comment:
@timgit I specified this to `7 days`. That introduces a breaking change from previous versions. If you want I can specify this as `12 hours` to be backwards compatible. WDYT?